bedrock/base/templates/includes/protocol/navigation/menu-firefox/browsers.html

```diff
@@ -0,0 +1,94 @@
+                <li><a href="https://support.mozilla.org/products/mobile/{{ referral|safe }}" rel="external noopener" data-link-name="Android Support" data-link-type="nav" data-link-position="subnav" data-link-group="browsers">{{ _('Android Support') }}</a></li>
+                <li><a href="{{ url('privacy.notices.firefox') }}" data-link-name="Mobile Privacy" data-link-type="nav" data-link-position="subnav" data-link-group="browsers">{{ _('Privacy') }}</a></li>
+              </ul>
+            </section>
+          </li>
+        </ul>
+        <ul>
+          <li>
+            <section class="mzp-c-menu-item mzp-has-icon">
+              <a class="mzp-c-menu-item-link" href="{{ url('firefox.enterprise.index') }}" data-link-name="Firefox for Enterprise" data-link-type="nav" data-link-position="topnav" data-link-group="browsers">
+                <svg class="mzp-c-menu-item-icon" width="24" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M20.289 15.5a1 1 0 0 1 1.842.78A11 11 0 1 1 7.6 1.912a1 1 0 0 1 .8 1.834A9 9 0 1 0 20.289 15.5zM23 12a1 1 0 0 1-1 1H12a1 1 0 0 1-1-1V2a1 1 0 0 1 1-1 11 11 0 0 1 11 11zm-4.636-6.364A9 9 0 0 0 13 3.056V11h7.944a9 9 0 0 0-2.58-5.364z" fill="#000" fill-rule="nonzero"/></svg>
+                {# L10n: please do not translate the product name "Firefox Browser" #}
+                <h4 class="mzp-c-menu-item-title">{{ _('Firefox Browser for Enterprise') }}</h4>
+                <p class="mzp-c-menu-item-desc">{{ _('Get unmatched data protection on the release cadence that suits your organization.') }}</p>
+              </a>
+            </section>
+          </li>
+          <li>
+            <section class="mzp-c-menu-item mzp-has-icon">
+              <a class="mzp-c-menu-item-link" href="https://mixedreality.mozilla.org/{{ referral|safe }}" rel="external noopener" data-link-name="Firefox Reality" data-link-type="nav" data-link-position="topnav" data-link-group="browsers">
+                <svg class="mzp-c-menu-item-icon" xmlns="http://www.w3.org/2000/svg" width="24" height="20" viewBox="0 0 24 20"><g fill="none" fill-rule="evenodd" stroke="#000" stroke-linecap="round" transform="translate(1 1)"><path stroke-linejoin="round" stroke-width="2" d="M8,18 C4,18 0,16 0,14 L0,4 C0,5.62640489 4,8 8,8 L8,18 Z"/><path stroke-linejoin="round" stroke-width="2" d="M22,18 C18,18 14,16 14,14 L14,4 C14,5.62640489 18,8 22,8 L22,18 Z" transform="matrix(-1 0 0 1 36 0)"/><path stroke-width="2" d="M0,4 C0,2 5,0 11,0 C17,0 22,2 22,4 L22,14 C22,12 17,11 11,11 C5,11 0,12 0,14 L0,4 Z"/></g></svg>
+                <h4 class="mzp-c-menu-item-title">Firefox Reality</h4>
+                <p class="mzp-c-menu-item-desc">{{ _('Strap on your goggles and step into the immersive web with Firefox’s VR browser.') }}</p>
```

Comment:
From @peiying2:

> Firefox as a brand must be left unchanged. This format Firefox's xyz will force some languages to declense the brand so it looks like Firefoxa or Firefoksa, to name a few, to be grammatically correct. This is for languages using Latin scripts. Can we change the wording to make it easier for localizer, like VR browser of Firefox. For other non-latin scripts, some face equally challenging situation.

I think something like "Firefox Browser for VR" could work? We should ask Natalie.